.github/workflows/_check_is_rust.yaml

```diff
@@ -0,0 +1,35 @@
+name: Check if app is based on Rust
+
+on:
+  workflow_call:
+    outputs:
+      is_rust:
+        description: Returns "true" if the app is using Rust SDK, else returns "false"
+        value: ${{ jobs.get-is-rust.outputs.is_rust }}
+
+jobs:
+  get-is-rust:
+    name: Check if app is based on Rust
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clone app repository
+        uses: actions/checkout@v3
+        with:
+          path: app-repository
+          submodules: recursive
+
+      - name: Check presence of Rust app file indicator
+        id: get-is-rust
+        run: |
+          if [[ -e app-repository/ledger_app.toml ]];
```

Comment:
I wonder if checking for `ledger_app.toml` file is enough to detect rust app. It's not mandatory to have this file is it ? For instance, https://github.com/LedgerHQ/app-avalanche does not have this file.

Reply:
It's currently mandatory for an app hat need to be deployed as a Rust app.
App Avalanche is a C app which embed some Rust code.